assets/src/settings-page/style.css

```diff
@@ -361,13 +364,13 @@ li.error-kept {
 	border-bottom-width: 1px;
 }
 
-#reader-themes-drawer {
+#reader-themes {
```

Comment:
Thanks for making this change. I was going to suggest it but I didn't want to nit pick… too much. 😊